inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/AuditServiceImpl.java

```diff
@@ -178,6 +181,11 @@ public Boolean refreshBaseItemCache() {
 
     @Override
     public Integer updateAuditSource(AuditSourceRequest request, String operator) {
+        String url = request.getUrl();
+        if (StringUtils.isNotBlank(url) && url.startsWith(MYSQL_JDBC_PREFIX)) {
```

Comment:
The audit source may not be just MySQL, it may also be clickhouse.So the URL may also be prefixed with ` jdbc: clickhouse`

Reply:
ok